.trybe/requirements.json

```
@@ -3,16 +3,49 @@
     "description": "1 - Crie um componente `<Header />`",
     "bonus": false
   }, {
-    "description": "2 - Crie um componente `<MovieList />`",
+    "description": "2 - Renderize um texto no `<Header />`",
     "bonus": false
   }, {
-    "description": "3 - Crie um componente `<MovieCard />`",
+    "description": "3 - Crie um componente `<MovieList />`",
     "bonus": false
   }, {
-    "description": "4 - Crie um componente `<Rating />`",
+    "description": "4 - Renderize componentes `<MovieCard />` dentro de `MovieList`",
```

```suggestion
    "description": "4 - Renderize componentes `<MovieCard />` dentro de `<MovieList />`",
```

Mudar também no README e no respectivo teste!